kuma/static/js/wiki-tags-edit.js

```diff
@@ -15,10 +15,12 @@
             var itemTexts = [];
 
             // Don't add list items we're going to remove
-            if(!isRemove) listItems.push(li);
+            if(!isRemove) {
+                listItems.push(li);
+            }
 
             // Cycling through each list item,
```

Comment:
Nit: Could this comment be more descriptive? Perhaps we simply do not need it?

Reply:
I didn't write or edit the comment, not sure why it shows up as a change. I'd like to leave it alone and just make the linter happy.